components/connector/docs/api/connectorapi.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: '3.0.0'
+info:
+  version: '1.0.0'
+  title: 'Compass Connector Resolver API'
+paths:
+  /health:
+    get:
+      summary: 'Returns health status.'
+      responses:
+        '200':
+          description: 'The service is up.'
+
+  /v1/tokens/resolve:
+    post:
+      parameters:
+      - in: header
+      name: Connector-Token
+        schema:
```

Comment:
Hey! I checked it in the Swagger Editor and got an error in this line. Is seems to be `Parser error - bad indentation of a mapping entry` here